### ISSUE TITLE
Unable to deploy with mongodb replset

### ISSUE THREAD

**Issue by bkleiner** [alerta/docker-alerta#41]
Hi,

Im trying to deploy this container with a mongodb replica set for HA as advised by the documentation.
The [docker-entrypoint script](https://github.com/alerta/docker-alerta/blob/master/docker-entrypoint.sh#L43) uses a mongodb client version 2.6 which needs a connection uri in the [form](https://docs.mongodb.com/v2.6/reference/program/mongo/):

`<replset>/<host0>,<host1>...`

If i set the `MONGO_URI` env var in such form to make the mongo client happy, alerta fails to connect to mongodb because of the invalid connection uri.

Setting `MONGO_URI` in alertad.conf doesn't help either because it seems the env var overwrites it. 

PS: Thanks! Really awesome project 👍

**Comment by bkleiner** [comment]
Maybe [this](https://github.com/str8labs/docker-alerta/commit/a2a6293c008adc0c832212acfcc671113c97586e) would be a solution, only set MONGO_ADDR if it wasn't set alreay, allowing the user to specify a seperate url for the mongo client.

**Comment by satterly** [comment]
Fixed by #43

**Comment by satterly** [comment]
You can easily test this yourself by checking out this repo and running ...
```
docker-compose -f docker-compose.yml -f docker-compose.mongo-replica.yml up
```
